backend/app/models/database.py

```
@@ -300,7 +317,108 @@ def _set_sqlite_pragma(dbapi_conn, connection_record):
+    column_list = ", ".join(shared_columns)
+    create_new_sql = str(
+        _render_create_table(Trade.__table__, bind, override_name="trades_new")
+    )
+
+    with bind.begin() as conn:
+        conn.execute(text("DROP TABLE IF EXISTS trades_new"))
+        conn.execute(text(create_new_sql))
+        conn.execute(
+            text(
+                f"INSERT INTO trades_new ({column_list}) "
+                f"SELECT {column_list} FROM trades"
```

Ruff (0.15.17)

[error] Possible SQL injection vector through string-based query construction

(S608)